graphql2/schema.graphql

```diff
@@ -2,7 +2,7 @@ type Query {
   phoneNumberInfo(number: String!): PhoneNumberInfo
 
   # Returns the list of recent messages.
-  debugMessages(input: DebugMessagesInput): [DebugMessage!]!
```

Comment:
This (and input/output) will need to stay identical for API compatibility reasons.

Add `messageLogs` as a net-new query.

For the GraphQL implementation, maybe just have the `debugMessages` query return the first page or something like that and mark it as deprecated, so we don't have the duplicate code.

Reply:
I thought about this once I was opening the draft- good call